ddtrace/contrib/grpc/patch.py

```diff
@@ -120,3 +229,19 @@ def _server_constructor_interceptor(wrapped, instance, args, kwargs):
         kwargs["interceptors"] = (interceptor,)
 
     return wrapped(*args, **kwargs)
+
+
+def _aio_server_constructor_interceptor(wrapped, instance, args, kwargs):
+    pin = Pin.get_from(grpc.aio.Server)
```

Comment:
```suggestion
    pin = Pin.get_from(GRPC_AIO_PIN_MODULE_SERVER)
```

did you also want to use the constant here?

Reply:
Thank you, at first I tried to replace the constants with `grpc.aio.XXX` and I guess this was left behind...